lib/es-models/bib.js

```diff
@@ -296,6 +297,10 @@ class EsBib extends EsBase {
       const day = second.slice(2, 4)
       dates.push(generateDateRangeFromSingleDate(first, month, day, rawMarc, type))
```

Comment:
Should all invocations of `generateDateRangeFromSingleDate` be wrapped in a try-catch? There may be date type conditions where we happen to know that's not likely, but the caller shouldn't know that.